coalib/processes/Processing.py

```diff
@@ -523,6 +557,7 @@ def process_queues(processes,
             control_elem, index = control_queue.get(timeout=0.1)
 
             if control_elem == CONTROL_ELEMENT.GLOBAL:
+                result_files.update(get_file_list(global_result_dict[index]))
```

Comment:
wouldn't it be good to simply store the results in the cache and yield them again next time?